scclasses/SCVim.sc

```diff
@@ -155,8 +156,24 @@ classvar <scvim_dir, <scvim_cache_dir,
         };
 
         postln("SCVim: processing help docs, this takes a little while....");
+        // regexp searching for filenames
+        glob_files = { arg folder, match;
+                var p;
+                PathName(folder).filesDo { arg x;
+                        if (match.matchRegexp(x.fileName)) {
+                                p = p.add(x.fileNameWithoutExtension);
+                                p = p.add(x.fullPath);
+                        }
+                };
+                p
+        };
+        postln("SCVim: Searching for all help files in " ++ Platform.resourceDir);
+        all_help_files =  Dictionary.newFrom(glob_files.value(Platform.resourceDir, "(?i)\\.(html|htm|rtf|schelp|scd)$"));
```

Comment:
@gusano yes! rtf should definitely be removed and never used.

html still has to be rendered from the schelp files. SCIDE does that on the fly when opening any specific help (but not all at once), and those to "~/.local/share/SuperCollider/Help" (on Linux).
If you only use the HelpBrowser through vim, html will not be rendered unfortunately.